save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/userprofile/UserProfileView.kt

```diff
@@ -371,6 +373,42 @@ fun ChildrenBuilder.renderLeftUserMenu(
     }
 }
 
+/**
+ * @param name
+ * @param header
+ */
+fun ChildrenBuilder.validateName(name: String?, header: HTMLAttributes<HTMLHeadingElement>) {
```

Comment:
may be rename to smth like `shortenTheNameIfNecessary`

Reply:
Done